diff --git forkSrcPrefix/Editor/Mono/EventWithPerformanceTracker.cs forkDstPrefix/Editor/Mono/EventWithPerformanceTracker.cs
index 6fb7bf4224651c6c19fecc9a00efbe205517a775..ceb9a98d12d233397e5bf479d82fd257689edd20 100644
--- forkSrcPrefix/Editor/Mono/EventWithPerformanceTracker.cs
+++ forkDstPrefix/Editor/Mono/EventWithPerformanceTracker.cs
@@ -419,7 +419,7 @@ namespace UnityEditor
                 m_Delegates = EventWithPerformanceTracker<T>.BuildDelegate(m_Scope, value);
             }
 
-            return new Invoker(ref m_Delegates);
+            return new Invoker(m_Delegates);
         }
 
         internal readonly ref struct Invoker
@@ -427,7 +427,7 @@ namespace UnityEditor
             private readonly EventWithPerformanceTracker.Entry m_Delegates;
             public Enumerator GetEnumerator() => new Enumerator(in this);
 
-            public Invoker(ref EventWithPerformanceTracker.Entry dlg)
+            public Invoker(EventWithPerformanceTracker.Entry dlg)
             {
                 m_Delegates = dlg;
             }
